### DISCUSSION TITLE
Best way to skip first and emit rest

### DISCUSSION THREAD

**Discussion by jacoobes** [ReactiveX/rxjs#7303] [Q&A] [answered]
i have some observables 
```ts
ready$ 
buildModules$
```
I want to chain these observables so that ready$ is called first always, and buildModules$ calls after.
In other words,  these are called sequentially, but the output of ready$ is ignored
I have this, and it works
```ts
ready$.pipe(
   switchMapTo(() => buildModules$)
)
```
But I feel like there could be a better way to tackle this. 
What do you guys recommend?

**Comment by demensky** [comment, marked as answer]
In the example below, the subscription to `buildModules$` will happen only after `complete` happens in `ready$`.  Events from `ready$` will be completely ignored.

```ts
concat(
  ready$.pipe(ignoreElements()),
  buildModules$,
)
```

**Comment by jacoobes** [comment]
thanks!
